packages/components/src/Filter/components/FilterPopover/FilterPopover.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { HTMLAttributes, useState } from "react"
+import { Options } from "@popperjs/core"
+import classnames from "classnames"
+import { usePopper } from "react-popper"
+import { OverrideClassName } from "@kaizen/component-base"
+import styles from "./FilterPopover.module.scss"
+
+export interface FilterPopoverProps
+  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
+  children: React.ReactNode
+  referenceElement: HTMLElement | null
+  popperOptions?: Partial<Options>
+}
+
+export const FilterPopover = ({
```

Comment:
Can this be a generic `Popover` to be (later) used in other places like `Menu`?

Reply:
We actually have a Popover component which is a little more like a tooltip.
This is certainly rather generic (if you consider the positioning has the offset and is only used in a context where it's `role="dialog"`), but we can't use the Popover name currently.
Since this is the AIO though, and this component is used by the Filter but not exported, we can rename and move it around later when we know what we want.